charts/s3gw/templates/tests/smoke-bucket-create.yaml

```diff
@@ -27,6 +27,6 @@ spec:
               value: {{ .Values.secretKey | quote }}
             - name: S3_HOSTNAME
               value:
-                '{{ .Chart.Name }}-svc.{{ .Release.Namespace }}.svc.cluster.local'
+                '{{ .Values.serviceName }}.{{ .Release.Namespace }}.{{ .Values.privateDomain }}.svc.cluster.local'
```

Comment:
Would this not expand to e.g. `mys3gw.mynamespace.svc.cluster.local.svc.cluster.local`? Should the `svc.cluster.local` at the end not be omitted or at least also a template like `{{ .Values.privateDomain }}`?

Reply:
you right, this is wrong, will fix.